neurons/miner.py

```diff
@@ -41,6 +35,7 @@ class Miner(BaseMinerNeuron):
 
     def __init__(self, config=None):
         super(Miner, self).__init__(config=config)
+        self.model = None
```

Comment:
why is this necessary?

Reply:
PEP8, class attributes should be instantiated in __init__

Reply:
i.e. make pycharm stop complaining